registrations/serializers.py

```diff
@@ -102,14 +103,28 @@ def _notify_contact_person(
     )
 
 
-def _validate_registration_enrolment_times(registration: Registration) -> None:
+def _validate_registration_enrolment_times(
+    registration: Registration, user: User
+) -> None:
     enrolment_start_time = registration.enrolment_start_time
     enrolment_end_time = registration.enrolment_end_time
     current_time = localtime()
 
     if enrolment_start_time and current_time < enrolment_start_time:
         raise ConflictException(_("Enrolment is not yet open."))
-    if enrolment_end_time and current_time > enrolment_end_time:
+
+    # Allow only a superuser or a registration admin to add signups to a closed registration.
+    if (
+        enrolment_end_time
+        and current_time > enrolment_end_time
+        and not (
+            user.is_authenticated
+            and (
+                user.is_superuser
+                or user.is_registration_admin_of(registration.publisher)
+            )
+        )
+    ):
```

Comment:
I started to wonder that should substitute users be allowed to add signups to a closed registration?

Reply:
I believe they should, and maybe also event admins who have created the registration?

Reply:
Implemented the changes to allow also a substitute user and a created by event admin to add signups to a closed registration.